### DISCUSSION TITLE
Log file etc for slinger

### DISCUSSION THREAD

**Discussion by sp222222** [GerryDazoo/Slinger#17] [General]
Gerry:

Me again, totally lost. 

Where are these logs to show the remote codes?

Where do we put the ?35 part  -- does it go on a certain line in the config.ini file or the remote.txt file?

I've noticed when I press the RT button for my 500 it powers off on each code 1,2,3, etc. What gives there? 

I did see that the new remote changed the number keys to green buttons now.

**Comment by GerryDazoo** [comment]
Can you give me a call tomorrow and I can walk you through it?

 

*	Gerry

 

From: sp222222 ***@***.***> 
Sent: Sunday, August 07, 2022 4:46 PM
To: GerryDazoo/Slinger ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [GerryDazoo/Slinger] Log file etc for slinger (Discussion #17)

 

Gerry:

Me again, totally lost.

Where are these logs to show the remote codes?

Where do we put the ?35 part -- does it go on a certain line in the config.ini file or the remote.txt file?

I've noticed when I press the RT button for my 500 it powers off on each code 1,2,3, etc. What gives there?

I did see that the new remote changed the number keys to green buttons now.

**Comment by sp222222** [comment]
Good there with the 500 but the solo I cannot get any remote codes to work.  solo streams fine otherwise. 

solo is on S-video input with PVR/DVR remote codes.

I will check my other solo later as I am trying to keep things in separate folders until I get each config and remote ini created.

thanks again Gerry

**Comment by GerryDazoo** [comment]
None of the buttons including power?!. All I can suggest is double check it’s working OK with slingplayer. Try remote code 0.

 

*	Gerry

**Comment by sp222222** [comment]
power works on 1 and 251

**Reply by sp222222** [reply]
solo Master Bedrm Got Streamer Control Message IR b'\x11\x00\x00\x00'
solo Master Bedrm Sending IR command b'\x11\x00\x00\x00'
08/08/2022, 13:49:23   connection from ('192.168.1.86', 59516)

Starting remote control stream hander for  ('192.168.1.86', 59516)
Remote Control Connected
Fixing 23 46 Host: 192.168.1.86:8082
Remote Type: 0 Src: 0 0x0 0x0 0x0
Button Clicked ImmutableMultiDict([('3', '11 : round'), ('Digits', '')])
127.0.0.1 - - [08/Aug/2022 13:49:23] "POST /Remote HTTP/1.1" 200 -
solo Master Bedrm Got Streamer Control Message IR b'\x0b\x00\x00\x00'
solo Master Bedrm Sending IR command b'\x0b\x00\x00\x00'

**Reply by sp222222** [reply]
I went back to version 2.06 of server and solo all buttons work. 
I have both 500 and solo streaming same time on same network segment.

**Reply by sp222222** [reply]
http://www.hifi-remote.com/forums/ 
under general there is a slingbox forum with bins and remotes.
it is still being updated with codes to use.
